aldryn_events/tests/frontend/unit/test.cl.events.js

```diff
@@ -30,7 +29,7 @@
             expect(Cl.events.calendar).toBeDefined();
         });
 
-        it('Cl.events.init() returns undefined', function () {
+        it('has Cl.events.init() to return undefined', function () {
```

Comment:
we can have nested describes
which means it would be better to do
```
describe('cl.events.js:', function () {
    // check the namespace and bla bla
    // check that methods do exist
    describe('.init()', function () {
         it('does this', functoin () { ... })
         // etc
    });
})